resources/util.sh

```diff
@@ -190,17 +210,17 @@ function installDefaultDockerRegistry() {
   export NEXUS_SERVER="http://localhost:8081/nexus"
 
   NEXUS_PASSWORD="${ADMINPW}" \
-    nexus-claim plan -i /defaultDockerRegistry.hcl -o "-" | \
-  NEXUS_PASSWORD="${ADMINPW}" \
-    nexus-claim apply -i "-"
+    nexus-claim plan -i /defaultDockerRegistry.hcl -o "-" |
```

Comment:
why was this line continuation removed? Is it superfluous?

Reply:
This does not look correct, good find!